npm/react/docs/recipes.md

```diff
@@ -34,7 +34,7 @@ module.exports = (on, config) => {
 }
 ```
 
-See example repo [bahmutov/try-cra-with-unit-test](https://github.com/bahmutov/try-cra-with-unit-test) or included example in the folder [examples/react-scripts](examples/react-scripts).
+See example repo [bahmutov/try-cra-with-unit-test](https://github.com/bahmutov/try-cra-with-unit-test) or included example in the folder [https://github.com/cypress-io/cypress/tree/develop/npm/react/examples/react-scripts](https://github.com/cypress-io/cypress/tree/develop/npm/react/examples/react-scripts).
```

Comment:
The display text for these links should still be the folder path (i.e. `examples/react-scripts`)

Reply:
Ahhh, find and replace fail.

Reply:
Fixed.